### ISSUE TITLE
Unable to publish config, small tweak.

### ISSUE THREAD

**Issue by meredevelopment** [edalzell/laravel-passwordless-login#51]
Whilst setting this up with Laravel 6, I wasn't able to publish the config. 

When I published with this `php artisan vendor:publish --tag=passwordless-login-config`:
Kept getting the error:
```
Can't locate path: <0>
Can't locate path: <1>
```

Had to change from this:

``` php
$this->publishes([
   __DIR__.'../config/config.php', config_path('laravel-passwordless-login.php'),
], 'passwordless-login-config');
```
to
``` php
$this->publishes([
    __DIR__.'/../config/config.php' => config_path('laravel-passwordless-login.php'),
], 'passwordless-login-config');
```

Works fine now. If you want this as a PR lemme know 👍🏼
Love your work.

**Comment by edalzell** [comment]
PR created

**Comment by edgrosvenor** [comment]
Got that merged. I appreciate the help.

**Comment by edalzell** [comment]
awesome! Thanks.

**Comment by dajoeberlei** [comment]
is it already merged? I still have the issue and im on 1.4 :-( i used @meredevelopment 's fix though! Thanks!

**Comment by edgrosvenor** [comment]
@dajoeberlei That's strange. It's been merged since 1.4.0. Glad you got it working.
